tests/_test_msui/test_msui.py

```diff
@@ -89,7 +89,7 @@ def test_milestone_url(self):
         with urlopen(self.window.milestone_url) as f:
             text = f.read().decode("utf-8")
         expected_version = __version__
-        pattern = rf'value="is:closed milestone:{re.escape(expected_version)}"'
+        pattern = rf'value="is:closed milestone:{re.escape(expected_version)} "'
```

Comment:
seems gh has changed the content. 

The test is then very fragile and needs a better solution

Reply:
https://github.com/Open-MSS/MSS/issues/2816